src/consumingMiddlewareAddon.ts

```diff
@@ -0,0 +1,354 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
THis file is really a copy of the notebookMiddlewareAddon but with the logic all reversed. If a request is for something that isn't a notebook cell, let it happen normally, otherwise do nothing.